### DISCUSSION TITLE
Show Tailscale Interface

### DISCUSSION THREAD

**Discussion by argie-w** [jamesog/tailscale-edgeos#25] [General]
Thank you for this guide. Is it possible to get the Tailscale interface to show on the main Dashboard?

**Comment by jamesog** [comment]
If you mean the EdgeOS web interface, then no.

Ubiquiti allow installing software packages but it's not exactly well supported, and I'm pretty sure that modifying core parts of the OS isn't possible or wouldn't persist.

**Reply by argie-w** [reply]
Ok got it thanks. It's just the wireguard package for EdgeOS shows as an interface so I thought it might be possible also for Tailscale.

**Reply by jamesog** [reply]
If that's an official package from Wireguard, perhaps they found a way to do it. You could always try opening an issue with Tailscale to see if they'd be willing to add more official support for EdgeOS. This is just an unofficial guide based on what they already packaged.
